native/core/src/execution/datafusion/expressions/sum_decimal.rs

```diff
@@ -127,19 +127,22 @@ impl AggregateUDFImpl for SumDecimal {
     fn reverse_expr(&self) -> ReversedUDAF {
         ReversedUDAF::Identical
     }
+
+    fn is_nullable(&self) -> bool {
+        // SumDecimal is always nullable because overflows can cause null values
```

Comment:
Wondering if this is true for ANSI.
It looks the previous code is also hardcoding true, but this may be a good time to file an issue if there is not yet.

Reply:
Good point. I filed https://github.com/apache/datafusion-comet/issues/961